src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserPostDao.java

```diff
@@ -54,4 +62,103 @@ public boolean add(UserPost post) {
         return isAdded;
     }
 
+    @Override
+    public boolean updateRecoveryUserToken(long userId, String recoveryToken) {
+        boolean isUpdate = false;
+        try (Connection connection = dataSource.getConnection();
+             PreparedStatement updateTokenStatement = connection.prepareStatement(UPDATE_RECOVERY_TOKEN)) {
+            updateTokenStatement.setString(1, recoveryToken);
+            updateTokenStatement.setLong(2, userId);
+            log.debug("Prepared statement {'preparedStatement':{}}.", updateTokenStatement);
+            int result = updateTokenStatement.executeUpdate();
+            if (result > 0) {
+                isUpdate = true;
+            }
+        } catch (PSQLException e) {
```

Comment:
Creating a table is neither an INSERT nor an UPDATE, so it's normal to receive 0 as no row(s) were affected. So it will be better additionally checking